assets/airports/katl.json

```diff
@@ -943,66 +914,133 @@
+                "KATL27R": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL28": [["CHPPR", "A130"], "MRCHH", "HIDRO", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"]
+            },
+            "draw": [
+                []
+            ]
+        },
+        "GLAVN1": {
+            "icao": "GLAVN1",
+            "name": "Glavin One",
+            "entryPoints": {
+                "JKSON": ["JKSON", "MADDX"],
+                "MGRIF": ["MGRIF", "DENST", "AAARN"],
+                "RUSSA": ["RUSSA", "RUTTH", "JKSON", "MADDX"]
+            },
+            "body": ["GLAVN", "STHRN"],
+            "rwy": {
+                "KATL8L": [["YABBA", "A90"], ["NAVVY", "A80+"], ["LARII", "A50+"], ["JAAJJ", "A50+"]],
+                "KATL8R": [["YABBA", "A90"], ["NAVVY", "A80+"], ["GPEAT", "A50+"], ["PEARI", "A50+"]],
+                "KATL9L": [["YABBA", "A90"], ["NAVVY", "A80+"], ["RYENN", "A70+"], ["AAKAY", "A60+"]],
+                "KATL9R": [["YABBA", "A90"], ["NAVVY", "A80+"], ["ANDIY", "A70+"], ["DFINS", "A60+"]],
+                "KATL10": [["YABBA", "A90"], ["NAVVY", "A80+"], ["^DROYD", "A80"], "#180"],
+                "KATL26L": ["HOGGG", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL26R": ["HOGGG", "RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL27L": ["HOGGG", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL27R": ["HOGGG", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL28": ["HOGGG", "EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"]
+            },
+            "draw": [
+                []
+            ]
+        },
+        "GNDLF2": {
+            "icao": "GNDLF2",
+            "name": "Gandalf Two",
+            "entryPoints": {
+                "BEORN": ["BEORN", ["STRDR", "A240+"], "GIMLY", "AZZOG"],
+                "COOUP": ["COOUP", "KHMYA", "CNDLR", ["FNLEY", "A240+"]],
+                "DRSDN": ["DRSDN"],
+                "ENNTT": ["ENNTT", ["GONDR", "A240+"]],
+                "FRDDO": ["FRDDO", "BLLBO", ["BGGNS", "A240+"]],
+                "GOLLM": ["GOLLM", ["ROHUN", "A240+"], "AZZOG"],
+                "KHMYA": ["KHMYA", "CNDLR", ["FNLEY", "A240+"]],
+                "ORRKK": ["ORRKK", "MORDR", "RAETH", ["NZGUL", "A240+"]],
+                "SHYRE": ["SHYRE", "FRDDO", "BLLBO", ["BGGNS", "A240+"]],
+                "STRDR": [["STRDR", "A240+"], "GIMLY", "AZZOG"]
+            },
+            "body": ["SMAWG", ["GNDLF", "A110+|S250"], ["HALRR", "A100+|S250"], ["SHULR", "A80"], "JNGLE", "^QUBIT", "#005"],
+            "rwy": {
+                "KATL8L": [],
+                "KATL8R": [],
+                "KATL9L": [],
+                "KATL9R": [],
+                "KATL10": []
+            },
+            "draw": [
+                []

```

Comment:
Why are we not drawing here? Would be nice for future plans, if STARs are drawn.